docs/src/administration/users.md

```diff
@@ -72,67 +60,27 @@ Available commands:
 * `platform user:role`
   * View or change a user's role
 
-For example, the following command would add `alice@example.com` with **Project Admin** role in the current project.
+ **Example**
+
+ Add `user1@example.com` to the project with Project Admin permissions:
 
 ```bash
-platform user:add alice@example.com -r admin
+platform user:add user1@example.com -r admin
 ```
+Once you add the user to the project, they receive an invitation email to confirm their details and a registration link.
+
+**Example**
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+To give User2 different levels of access depending on the environment type in the current project:
 
-To give Bob different levels of access depending on the environment type, in the current project:
- 
 -  **Viewer** role to the **Production environment**
 -  **Contributor** role to **all Staging type environments**
 -  **Admin** role to **all Development type environments**
 
 You would run:
 
 ```bash
-platform user:role bob@example.com -r production:viewer -r staging:contributor -r development:admin
+platform user:role user2@example.com -r production:viewer -r staging:contributor -r development:admin
 ```
 
 Use `platform list` to get the full list of commands.
```

Comment:
I think this is unnecessary (it's true anytime the CLI is run, so I wouldn't repeat it).

I would put in an expected outcome -- does the user get an email this time?